package/tests/testCOUNT.py

```diff
@@ -45,7 +45,7 @@ def test_runTestWithParams(self):
                 totPROC      = prmonJSON["Max"]["nprocs"] 
                 totTHREAD    = prmonJSON["Max"]["nthreads"]
                 expectPROC   = procs
-                expectTHREAD = procs*threads 
+                expectTHREAD = procs*(threads+1)
```

Comment:
I think the issue here is in the burner, which is actually written so that it *spawns* an additional `n` threads, leaving one parent thread that doesn't run a CPU burner, but just waits for its worker threads to finish (hence then `n+1` total threads).

I think it's more sensible to fix the burner to really use `n` threads, instead of `n+1`.